core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1070,9 +1070,9 @@ public void terminateWorkflow(Workflow workflow, String reason, String failureWo
 				});
 				List<String> urns = (List<String>) map.get("urns");
 				for (String urn : urns) {
-					if (urn.contains("orderid:")) {
+					if (urn.contains("orderid:") || urn.contains("order:")) {
 						orderId = urn.substring(urn.lastIndexOf(':') + 1);
```

Comment:
We have a utility method that we use in couple of places.. may be we should move all that into a util class and have these in there.. check StatusEventPublisher.getJobId